.github/workflows/mark-and-close-stale-issues.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          stale-issue-message: 'This issue was marked as stale and going to be closed soon'
+          stale-pr-message: 'This PR was marked as stale and going to be closed soon'
+          days-before-stale: 15
```

Comment:
I left this feedback on the issue, but can we start with 30 plx?

Reply:
yep, no problem, that's why I added this parameter in the first place

Reply:
done